baseapp/baseapp.go

```diff
@@ -244,7 +244,8 @@ func (app *BaseApp) MountKVStores(keys map[string]*storetypes.KVStoreKey) {
 // MountTransientStores mounts all transient stores to the provided keys in
 // the BaseApp multistore.
 func (app *BaseApp) MountTransientStores(keys map[string]*storetypes.TransientStoreKey) {
-	for _, key := range keys {
+	keyL := maps.Values(keys)
```

Comment:
> [Values returns the values](https://pkg.go.dev/golang.org/x/exp/maps#Values) of the map m. **The values will be in an indeterminate order.**

Does this change solve a problem?

Reply:
It does.

Reply:
What problem does it solve? Is it to silence the linter warnings about nondeterinistic iteration order? This change does seem to silence the linter, but it doesn't change the behavior, right? The order is still nondeterministic.